src/components/Person/Person.jsx

```diff
@@ -1 +1,14 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+      {isMarried ? <p className="Person__partner">{`${partnerName} is my ${partner}`}</p> : <p className="Person__partner">I am not married</p> }
```

Comment:
My formatting is approximate because it's hard to edit on GitHub, but please do it like this
```suggestion
      <h2 className="Person__name">
        {`My name is ${name}`}
      </h2>
      
      {age && 
         <p className="Person__age">
            {`I am ${age}`}
         </p>}
         
      {isMarried 
         ? <p className="Person__partner">
               {`${partnerName} is my ${partner}`}
            </p> 
          : <p className="Person__partner">
             I am not married
            </p> }
```